sherpa/utils/akima.py

```diff
@@ -47,8 +47,8 @@
 Requirements
 ------------
 
-* `Python 2.7 or 3.2 <http://www.python.org>`__
-* `Numpy 1.6 <http://numpy.scipy.org>`__
+* `Python 2.7 or 3.2 <https://www.python.org>`__
+* `Numpy 1.6 <https://numpy.scipy.org>`__
```

Comment:
connection to https://numpy.scipy.org times out but https://scipy.org/ works

Reply:
Since this lists dependencies, it's numpy.org now that we want. But numpy 1.6 is of course years out of date, si this is more of a historical record now anyway.

Reply:
Perhaps we could change it to 'https://docs.scipy.org/doc/numpy-1.6.0/reference/'  which points to the old numpy 1.6 documentation (and dates it to May 15, 2011) and makes the association of numpy and scipy
.

Reply:
It doesn't actually use scipy though, it only uses numpy. So it really depends on who our audience is: A developer with a historical interest or a user/developer who gets stuck, and reads the code to see what dependences are listed to get it to run today. I think neither will ever happen in practice, so I don't really care; either is fine with me.

(I'm not going back to test that now or go back through git blame, but it's entirely possible that this code would not run with numpy 1.6 anymore. We changed a lot of file to make them numpy 2.0 compatible and I don't think it's worth the time to check how far back this one will work.)

Reply:
I am good with merging this but the github API is currently stuck 'Checking for the ability to merge automatically...'

Reply:
Let me manually rebase and force push. That should trigger it again. I won't do any changes, so we don't have to wait (and waste the electricity) for the tests to run through again. (I'll see if the `[skip CI]` still works.)

Reply:
Skipped GH checked, it's still trying to build RTDs (not sure how to skip that). It passed before and I don't see who that would have changed, but your call if you want to wait for that.